spec.bs

```diff
@@ -405,6 +416,12 @@ fetch(issueRequest);
 To <dfn>append private state token issue request headers</dfn> given a [=/request=] |request|, run the following steps:
 
  1. If |request|'s [=request/client=] is not a [=secure context=], return.
+ 1. If |type| is not specified, throw a "{{TypeError}}" {{DOMException}}.
+ 1. If |type| is not `"private-state-token"`, throw a "{{TypeError}}" {{DOMException}}.
```

Comment:
```suggestion
 1. If |type| is not `"private-state-token"`, throw a {{TypeError}}.
```

TypeError is a simple exception, see https://webidl.spec.whatwg.org/#idl-exceptions

Reply:
(same below)